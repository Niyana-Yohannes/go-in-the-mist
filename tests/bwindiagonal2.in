place A1
place S1
place B2
place S2
place C3
place S3
place D4
place S4
place E5
