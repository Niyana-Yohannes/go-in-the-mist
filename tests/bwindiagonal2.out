Black wins!
A1S1B2S2C3S3D4S4E5
Thank you for playing!
